app/models/user.rb

```diff
@@ -10,7 +10,7 @@ class User < ApplicationRecord
   has_secure_password
   validates :password, presence: true, length: { minimum: Settings.validates.password.minimum }
 
-  def User.digest(string)
+  def User.digest string
```

Comment:
Define class method using the following syntax by following style guide [https://github.com/framgia/coding-standards/blob/master/eng/ruby/standard.md](https://github.com/framgia/coding-standards/blob/master/eng/ruby/standard.md)
``
class << self
  def method_name
    # method body 
  end 
end 
``